Source/Svg.csproj

```diff
@@ -1,393 +1,145 @@
-    <DebugType>Full</DebugType>
-    <Optimize>false</Optimize>
-    <OutputPath>..\..\vvvv\public\common\src\thirdparty\</OutputPath>
-    <DefineConstants>TRACE;DEBUG;REFLECTION</DefineConstants>
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
-    <CodeAnalysisRuleSet>AllRules.ruleset</CodeAnalysisRuleSet>
-    <DebugSymbols>true</DebugSymbols>
-    <DocumentationFile>
-    </DocumentationFile>
-    <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
-    <NoWarn>1591</NoWarn>
-  </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
-    <DebugType>PdbOnly</DebugType>
-    <Optimize>True</Optimize>
-    <OutputPath>bin\Release\</OutputPath>
-    <DefineConstants>TRACE</DefineConstants>
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
-    <CodeAnalysisRules>
-    </CodeAnalysisRules>
-    <CodeAnalysisRuleSet>AllRules.ruleset</CodeAnalysisRuleSet>
-    <DebugSymbols>false</DebugSymbols>
-    <DocumentationFile>bin\Release\Svg.XML</DocumentationFile>
+    <IsPackable>true</IsPackable>
+    <Company></Company>
+    <Authors>davescriven,jvenema,owaits,ddpruitt,Ralf1108,Tebjan Halm,and others</Authors>
+    <PackageId>Svg</PackageId>
+    <Product>SVG Rendering Library</Product>
+    <Summary>SVG Rendering Library</Summary>
+    <Description>
+      Public fork of the C# SVG rendering library on codeplex: https://svg.codeplex.com/
+      This started out as a minor modification to enable the writing of proper SVG strings. But now after almost two years we have so many fixes and improvements that we decided to share our current codebase to the public in order to improve it even further.
+      So please feel free to fork it and open pull requests for any fix, improvement or feature you add.
+      License: Microsoft Public License: https://svg.codeplex.com/license
+    </Description>
+    <Copyright>Copyright © vvvv.org</Copyright>
+    <Tags>svg, vector graphics, rendering</Tags>
+    <ProjectGuid>{886A98C5-37C0-4E8B-885E-30C1D2F98B47}</ProjectGuid>
+    <DocumentationFile>$(OutputPath)Svg.XML</DocumentationFile>
+    <Configurations>Debug;Release</Configurations>
     <NoWarn>1591</NoWarn>
-  </PropertyGroup>
-  <PropertyGroup>
     <SignAssembly>true</SignAssembly>
-  </PropertyGroup>
-  <PropertyGroup>
     <AssemblyOriginatorKeyFile>svgkey.snk</AssemblyOriginatorKeyFile>
+    <AssemblyVersion>2.4.5.0</AssemblyVersion>
+    <FileVersion>2.4.5.0</FileVersion>
+    <Version>2.4.5</Version>
+    <PackageTags>svg, vector graphics, rendering</PackageTags>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <GenerateDocumentationFile>false</GenerateDocumentationFile>
+    <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
+    <PackageReleaseNotes>
+Supports multiple targets v3.5 thru to dotnetcore2.2. 
+NetStandard does not fully support the Drawing2D package - so has been left out.

```

Comment:
Again, you are probably correct. I did make an effort to get it working, but rather than spend time hacking a solution, I thought it better to wait for the platform to catch up.
My main focus was to amend the CI/CD process to support multiple targets. Once netstandard catches up, it should simply slot in.

Reply:
Agreed. Maybe we don't have to wait long for that.